tests/hotplug/instancetype.go

```diff
@@ -124,12 +121,7 @@ var _ = Describe("[sig-compute][Serial]Instance Type and Preference Hotplug", de
 		maxInstancetype, err = virtClient.VirtualMachineInstancetype(vmi.Namespace).Create(context.Background(), maxInstancetype, metav1.CreateOptions{})
 		Expect(err).ToNot(HaveOccurred())
 
-		// TODO - Move this into libvmi as a VMOption for the above NewVirtualMachine call
-		vm.Spec.Instancetype = &virtv1.InstancetypeMatcher{
-			Name: originalInstancetype.Name,
-			Kind: instancetypeapi.SingularResourceName,
-		}
-
+		vm = libvmi.NewVirtualMachine(vmi, libvmi.WithRunStrategy(virtv1.RunStrategyAlways), libvmi.WithInstancetype(originalInstancetype.Name))
```

Comment:
nit: I think that with this change we can remove the global `vm` var. WDYT?

Reply:
ACK done, also removed some previously missed cruft that can be dropped now that we are using `WithInstancetype`.